src/Kernel32.Shared/Kernel32Extensions.cs

```diff
@@ -19,14 +19,15 @@ public static partial class Kernel32Extensions
         /// Gets the text associated with a <see cref="Win32ErrorCode"/>.
         /// </summary>
         /// <param name="error">The error code.</param>
+        /// <param name="dwLanguageId">The language identifier for the message</param>
         /// <returns>The error message. Or <c>null</c> if no message could be found.</returns>
-        public static unsafe string GetMessage(this Win32ErrorCode error)
+        public static unsafe string GetMessage(this Win32ErrorCode error, int dwLanguageId = 0)
```

Comment:
yes, that could be an idea. We could also offer friendly code generated overloads ;-)